src/test/java/net/coru/kloadgen/input/avro/AvroSubjectPropertyEditorTest.java

```diff
@@ -44,9 +52,44 @@ public void iterationStart(@Wiremock WireMockServer server) throws IllegalAccess
     JMeterContextService.getContext().getProperties().put(SCHEMA_REGISTRY_USERNAME_KEY, "foo");
     JMeterContextService.getContext().getProperties().put(SCHEMA_REGISTRY_PASSWORD_KEY, "foo");
 
-    AvroSerializedConfigElement avroSerializedConfigElement = new AvroSerializedConfigElement("AvroSubject", Collections.emptyList(), null);
+    AvroSerializedConfigElement avroSerializedConfigElement = new AvroSerializedConfigElement("avroSubject", Collections.emptyList(), null);
     JMeterVariables variables = JMeterContextService.getContext().getVariables();
     avroSerializedConfigElement.iterationStart(null);
     assertThat(variables).isNotNull();
+    assertThat(variables.get("SampleEntity")).isNotNull();
```

Comment:
Not so bad, but check [here](https://github.com/joel-costigliola/assertj-examples/blob/master/assertions-examples/src/test/java/org/assertj/examples/MapAssertionsExamples.java) there are nice and more elegant ways to assert that.